docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
-levels of scale, price, performance and flexibility, just by running simple python
-scripts from their laptop.
+## Available Computational Resources
 
-## Available Computational Options
-As of version 0.4, Inductiva is able to send computational loads to Google Cloud
-Platform (GCP). Therefore, the simulations you start using the API will run on one
-VM or several VMs (if you start an MPICluster) that live on GCP. We make available
-VMs of the following families:
+In the current release, version 0.4, Inductiva supports dispatching computational 
+workloads to the Google Cloud Platform (GCP). This means that the simulations initiated through our API are executed on one or more virtual machines (VMs) hosted on GCP. 
 
-**Compute Optimized:**
+The VM families made available by Inductiva on Google Cloud
+Platform (GCP) include:
 
-- [C2](https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types)
-- [C2D](https://cloud.google.com/compute/docs/compute-optimized-machines#c2d_series)
-- [H3](https://cloud.google.com/compute/docs/compute-optimized-machines#h3_series)
 
-**General Purpose:**
+````{eval-rst}
+.. tabs::
 
- - [N1]( https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines)
- - [N2](https://cloud.google.com/compute/docs/general-purpose-machines#n2_series)
- - [N2D](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines)
- - [C3](https://cloud.google.com/compute/docs/general-purpose-machines#c3_series)  
- - [C3D](https://cloud.google.com/compute/docs/general-purpose-machines#c3d_series)
+   .. tab:: Compute-optimized Machines
 
-Virtual Machines from these families typically come in three variants, with increasing amounts of RAM per vCPU:
+      - [C2](https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types)
+      - [C2D](https://cloud.google.com/compute/docs/compute-optimized-machines#c2d_series)
+      - [H3](https://cloud.google.com/compute/docs/compute-optimized-machines#h3_series)
 
-- **highcpu** - 2 Gb per vCPU 
-- **standard** - 4 Gb per vCPU
-- **highmem** - 8 Gb per vCPU
+   .. tab:: General-purpose Machines
 
-This means you can start MachineGroups, ElasticMachineGroups and MPIClusters with any of these VMs. For example, here is how you start a MachineGroup with beefy C3D general machines:
+       - [N1]( https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines)
+       - [N2](https://cloud.google.com/compute/docs/general-purpose-machines#n2_series)
+       - [N2D](https://cloud.google.com/compute/docs/general-purpose-machines#n2d_machines)
+       - [C3](https://cloud.google.com/compute/docs/general-purpose-machines#c3_series)  
+       - [C3D](https://cloud.google.com/compute/docs/general-purpose-machines#c3d_series)
+
+````
+
+The VMs within these families are categorized into three types, based on the RAM-to-vCPU ratio:
+
+- **highcpu** -  Offers 2 GB of RAM per vCPU, suited for CPU-intensive tasks.
```

Comment:
Here, I think we need to refactor the technical description, just adding this as a prompt for me to clean on a second iteration.